optimum/bettertransformer/models/base.py

```diff
@@ -128,7 +130,10 @@ def validate_bettertransformer(self):
             )
 
     def forward_checker(self, *args, **kwargs):
-        if torch.is_autocast_enabled() or torch.is_autocast_cpu_enabled():
+        nested_tensors_have_fp16_backend = packaging.version.parse(torch.__version__) >= packaging.version.parse(
```

Comment:
maybe this one can be in the `__init__`?

Reply:
moved it to `import_utils.py`